docs/installation.rst

```diff
@@ -76,6 +76,12 @@ suited for your deployment.
     META_SITE_PROTOCOL = 'http'
     META_USE_SITES = True
 
+* For meta tags support enable the needed types::
```

Comment:
Extra colon i.e. `::` instead of `:`.

Reply:
the `::` is used to render the following lines with code formatting (see similar formatting in the lines above)

Reply:
Ah, okay. I've been working with Markdown all weekend, and didn't notice that this `.rst`—not my realm 😅.

Reply:
:joy: